src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
You **don't need to create new "Random" object all the time** and it also will take more memory, so, you can add Random object as field of the ColorSupplier class. <br/>
AND: You also have to make the method non-static in order it to have access to "random" variable which is non-static. 
```suggestion
    private Random random = new Random();

    public String getRandomColor() {
        int index = random.nextInt(Colors.values().length);
```